### PR TITLE
[release] v1.1.0

### PR DESCRIPTION
**Pull request by benja-wu**

### CODE REVIEW

**Review by localvar — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ MKFILE_DIR := $(dir $(MKFILE_PATH))
 RELEASE_DIR := ${MKFILE_DIR}bin
 
 # Version
-RELEASE?=1.0.1
+RELEASE?=1.1.0
```
Comment (xxx7xxxx) [inline Makefile:14]: Make it consistent: v1.1.0

Comment (benja-wu) [reply Makefile]: Updated

Comment (localvar) [reply Makefile]: the docker tag becomes v1.1.0, is it right?

Comment (benja-wu) [reply Makefile]: In https://hub.docker.com/repository/docker/megaease/easegress, its tags are `v1.0.1`, `v1.0.0`.

Comment (xxx7xxxx) [reply Makefile]: Yes, please remember update tag `server-sidecar`.

**Review by xxx7xxxx — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (localvar) [reply Makefile]: I checked several other applications, they don't have a `v` in the version tag.

Comment (benja-wu) [reply Makefile]: > Yes, please remember update tag `server-sidecar`.

Sure. I will update it after we finished this releasing process.

Comment (xxx7xxxx) [reply Makefile]: https://semver.org/#is-v123-a-semantic-version
https://github.com/etcd-io/etcd/releases

Comment (benja-wu) [reply Makefile]: > I checked several other applications, they don't have a `v` in the version tag.

Emm. But we had already launched `v1.0.0`, and `v1.0.1`. If we chose to trim this `v` from the version tag, will it become inconsistent?

**Pull request merged**